Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python:3.11.6-alpine3.18
+
+ENV PYTHONDONTWRITEBYTECODE=1
+
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR /app
+COPY requirements.txt .
+RUN pip install --no-cache-dir -r requirements.txt
+
+COPY . .
+
+RUN adduser \
+    --disabled-password \
+    --no-create-home \
+    django-user
```

Comment:
The `adduser` command creates a user without a home directory or password, which is good for security. However, ensure that the application does not require a home directory for the `django-user` to function correctly.